lib/mpool/src/mclass.c

```diff
@@ -323,6 +340,9 @@ mclass_to_mcid(enum mpool_mclass mclass)
         case MP_MED_STAGING:
             return MCID_STAGING;
 
+        case MP_MED_PMEM:
+            return MCID_PMEM;
+
         default:
```

Comment:
Alex would probably say remove the default case here and mcid_to_mclass() in order for the compiler to warn about missing branch.